src/main/java/com/odysseusinc/arachne/executionengine/service/impl/RuntimeServiceImpl.java

```diff
@@ -188,6 +192,32 @@ public Future analyze(AnalysisSyncRequestDTO analysis, File file, StdoutHandlerP
         });
     }
 
+    private void prepareRDist() throws IOException, InterruptedException {
+
+        final String DIR_PREFIX = "rdist";
+
+        String tmpDir = MoreObjects.firstNonNull(System.getProperty("java.io.tmpdir"), "/tmp");
```

Comment:
I don't believe `/tmp` is a good place to store long living content. As to me it's better to use smth like `/var/arachne/cache` or similar.